styles.css

```diff
@@ -0,0 +1,370 @@
+
+* {
+  box-sizing: border-box;
+}
+h1 {
+  font-family: 'martel', serif;
+  color: rgb(102, 153, 0);
+  font-size: 2vw;
+  font-style: italic;
+  position: absolute;
+  top: 60%;
+  left: 5%;
+}
+h2, h3 {
+  font-family: 'martel', serif;
+  color: MidnightBlue;
+}
+h4 {
+  font-family: 'martel';
+  color: darkSlateblue;
+}
+h6 {
+  font-family: 'martel', serif;
+  color: MidnightBlue;
+  font-size: 1rem;
+}
+a {
+font-family: 'martel', serif;
+font-size: 1rem;
+}
+h5 {
+  font-family: 'martel', san-serif;
+  color: white;
+  font-size: .8rem;
+}
+p  {
+  font-family: 'martel', serif;
+  font-size: .8rem;
+}
+.form {
+  position: relative;
+  font-family: 'martel' , sans-serif;
+  margin-left: 25%;
+}
+.logo {
+  height: 150px;
+  top: 25%;
+  position: absolute;
+  margin-left: -25%;
+}
+.background {
+  height: 100%;
+  background-image: url("./images/header-image-narrow.jpg");
+}
+header {
+  position: relative;
+  background: linear-gradient(45deg, yellow, black 30%);
+}
+nav {
+  position: relative;
+  padding: 1rem;
+  background-color: lightGray;
+  color: rgb(17, 64, 105);
+}
+aside {
+    width: 20%;
+    float: right;
+    margin-right: 15%;
+    margin-left: 4%;
+  }
+.main {
+    width: 50%;
+    margin-left: 20%;
+  }
+.main-dogs {
+  width: 60%;
+  margin-left: 20%;
+}
+.footnav{
+  background-color:rgb(17, 64, 105) ;
+  color: white;
+}
+.checkout {
+  color: white;
+  position: absolute;
+  top: 10%;
+  right: 10%;
+}
+.cart {
+  width: 2.5rem;
+}
+.man-dog {
+  padding-top: 5%;
+  margin-right: 15%;
+  display: inline-block;
+  width: 50%;
+}
+.grid-container {
+  display: grid;
+  grid-template-columns: repeat(3, 1fr);
+  grid-gap: 10px;
+  }
+.grid-container-2 {
+    display: grid;
+    grid-template-columns: repeat(4, 1fr);
+    grid-gap: 20px;
+    }
+.grid-container > div {
+  width: 100%;
+  padding: 10px;
+  margin-top: 10px;
+  align-items: center;
+}
+.grid-container-2 > div {
+  width: 100%;
+  padding: 10px;
+  margin-top: 10px;
+  align-items: center;
+}
+col-1 > div {
+  padding: 10px;
+}
+.col-1 {
+    display: inline-grid;
+    max-width: 90%;
+    min-width: 10%;
+    box-shadow: 0 4px 8px 0 rgba(0, 0, 0, 0.2);
+    padding: 2px;
+    text-align: left;
+    background-color: white;
+    border: 1px;
+    border-color: lightGray;
+  }
+[class*="col-"] {
```

Comment:
Your col classes could be making better use of their inner white space, there is alot of margin white space around the p and h2,h3 elements from the user stylesheets(default browser settings) try and explicitly set them so the look the same on all machines. 

By the same note, the body element has some margins being applied by the default browser style. I would set them to 0 explicitly.